nativelink-util/tests/evicting_map_test.rs

```diff
@@ -523,6 +531,7 @@ async fn get_evicts_on_time() -> Result<(), Error> {
             max_seconds: 5,
             max_bytes: 0,
             evict_bytes: 0,
+            eviction_grace_period_seconds: 0,
```

Comment:
Nit: These could use `..EvictionPolicy::default()` and stop needing to add new default fields